gui/window.h

```diff
@@ -179,6 +180,7 @@ private slots:
     QProcessEnvironment setMacEnv();
     QStackedWidget *stackedWidget;
     bool isBasicView;
+    void delay();
```

Comment:
in addition to the the delay how about show a progress bar or "Loading .." message and wait till it actually it finishes it

Reply:
Good idea! Added
![Screen Shot 2022-04-28 at 11 58 33 AM](https://user-images.githubusercontent.com/44844360/165828471-23dd8d33-ede8-4f61-a49c-ac90bee936ca.png)